htmlcov/status.json

```diff
@@ -1,743 +1,1479 @@
 {
-  "format": 2,
```

Comment:
Is this file intended to be committed?  This looks like testing coverage output, although I may be wrong about that.

Reply:
Yeah, that's test coverage, I'll remove that.

Reply:
@jsf9k I removed the html coverage in another commit and merged it into this branch to reduce the amount of files changed to look at. So are html coverage is no longer being commited.